ironfish/src/rpc/adapters/socketAdapter/socketAdapter.ts

```diff
@@ -193,6 +197,21 @@ export abstract class RpcSocketAdapter implements IRpcAdapter {
 
       const message = result.result.data
 
+      if (this.router == null || this.router.server == null) {
+        this.emitResponse(client, this.constructUnmountedAdapter())
+        return
+      }
+
+      // Authentication
```

Comment:
```suggestion
```

I don't think we need this comment since it's fairly obvious what this is doing.

Reply:
I will keep the comment, the logic for onClientData is already very bloated with a lot of if statement. This help the code readability.